### PR TITLE
Include expect-playwright

### LINKED ISSUE #579: 📦Bundle expect-playwright with qawolf
- Install it as a dependency
- Auto-configure it with [js,ts]-jest.config.js
- Add assertion to e2e test
- Update add assertions doc

### PR DESCRIPTION
**Pull request by flaurida**
* resolves #579

### CODE REVIEW

**Comment by flaurida** [comment]
Will keep this PR in draft state until https://github.com/playwright-community/expect-playwright/issues/16 resolved.

File: docs/docs/add_assertions.md [hunk 1]
```diff
@@ -7,26 +7,28 @@ In this guide we show you how to add assertions to your tests. We assume that yo
...
 
 - Use [watch mode](edit_a_test#watch-mode) to automatically re-run your tests on save
 - The [interactive REPL](use_the_repl) lets you try out assertions while creating tests
 
-## Use the Playwright API
+## Create assertions
 
 In this guide, we'll add assertions to a test on [TodoMVC](http://todomvc.com/examples/react).
 
 You can add assertions as you create your test, since the [test code is generated](create_a_test#review-test-code) as you use your application. The [interactive REPL](use_the_repl) can be helpful in trying out code.
```
Comment (jperl) [inline docs/docs/add_assertions.md:27]: I would replace the line about the repl and say edit mode can be helpful.

File: docs/docs/add_assertions.md [hunk 2]
```diff
@@ -7,26 +7,28 @@ In this guide we show you how to add assertions to your tests. We assume that yo
...
-
+// expect-playwright
+await expect(page).toHaveText('Clear completed');
+await expect(page).not.toHaveSelector('.todo-list li');
+// Playwright API
 await page.waitFor(() => !document.querySelector('.todo-list li'));
 ```
 
 - Use [watch mode](edit_a_test#watch-mode) to automatically re-run your tests on save
 - The [interactive REPL](use_the_repl) lets you try out assertions while creating tests
```
Comment (jperl) [inline docs/docs/add_assertions.md:21]: We should stop recommending repl, and instead recommend edit mode.

Comment (jperl) [reply docs/docs/add_assertions.md:21]: Also we should only recommend one way of doing assertions, using expect-playwright.

File: docs/docs/add_assertions.md [hunk 3]
```diff
@@ -7,69 +7,73 @@ In this guide we show you how to add assertions to your tests. We assume that yo
...
 - Use [watch mode](edit_a_test#watch-mode) to automatically re-run your tests on save
-- The [interactive REPL](use_the_repl) lets you try out assertions while creating tests
 
-## Use the Playwright API
+## Create assertions
+
+In this guide, we'll add assertions to our test on [TodoMVC](http://todomvc.com/examples/react).
 
-In this guide, we'll add assertions to a test on [TodoMVC](http://todomvc.com/examples/react).
+We recommend adding assertions with the [expect-playwright library](https://github.com/playwright-community/expect-playwright), which makes it easy to write assertions with Playwright and Jest. QA Wolf includes this library by default, so you do not need to install anything to use it.
```
Comment (jperl) [inline docs/docs/add_assertions.md:23]: "We recommend adding assertions with the [expect-playwright library](https://github.com/playwright-community/expect-playwright), which auto-waits for assertions."

**Comment by jperl** [comment]
LGTM after next https://github.com/playwright-community/expect-playwright release

**Comment by jperl** [comment]
Thanks @mxschmitt for building such an awesome package! 🔥 🔥

**Pull request merged**